packages/runtime/test/integration/group-inner-transaction.ts

```diff
@@ -74,4 +76,89 @@ describe("Group inner transaction", function () {
 		// spend 5000 micro Algo
 		assert.equal(contractBalance, contractAcc.balance() + 5000n);
 	});
+
+	describe("Number inner transaction limitation", function () {
+		let secondAppInfo: AppInfo;
+		this.beforeEach(() => {
+			secondAppInfo = runtime.deployApp(
+				alice.account,
+				{
+					appName: "secondApp",
+					metaType: types.MetaType.FILE,
+					approvalProgramFilename: "limit-number-txn.py",
```

Comment:
For approval, we are passing pyteal but for the clear program, we are passing teal file. Is this expected?